metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -0,0 +1,26 @@
+=== windows perfmon MetricSet
+
+This is the perfmon metricset of the module windows.
+
+[float]
+=== Features and configuration
+
+Metricset to collect performance counters.
+Example configuration:
+```
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 10s
+  perfmon.counters:
+    - group: "processor"  
+      collectors:
+        - alias: "processor_perfomance"
```

Comment:
Thanks for sharing the event output: https://github.com/elastic/beats/pull/3758#issuecomment-287699129 Based on the output I was thinking if we should this `key` or `field` instead of `alias`? 

Do we need the "group" part, or could we just let the user define `processor. processor_perfomance ` and we would have the same outcome? Are there some additional functionalities of the groups except for the namespacing?

Other thing I'm thinking about is how far we can "align" the configs of `jmx` and `perfmon` as they try to tackle a similar problem. Here some examples: https://github.com/elastic/beats/blob/master/metricbeat/module/jolokia/jmx/_meta/docs.asciidoc Probably it is best to unify in a second step and first get your PR in ;-)